pr2eus/robot-interface.l

```diff
@@ -1284,9 +1284,10 @@ Return value is a list of interpolatingp for all controllers, so (null (some #'i
   (:go-stop (&optional (force-stop t))
    (when joint-action-enable
      (send move-base-action :cancel-all-goals)
-     (prog1
-         (send move-base-trajectory-action :cancel-all-goals)
-       (if force-stop (send self :go-velocity 0 0 0)))
+     (when move-base-trajectory-action
+       ;; some robots (e.g. fetch) don't have move-base-trajectory-action
+       (send move-base-trajectory-action :cancel-all-goals))
+     (if force-stop (send self :go-velocity 0 0 0))
```

Comment:
With your code, `:go-stop` returns `(send self :go-velocity 0 0 0)` result as return value when `force-stop` is `t`, but `:go-stop` always returns `nil` when `force-stop` is `nil` because `(if nil t) ;; return nil`.

Reply:
so the correct code might be like this.
```lisp
  (:go-stop (&optional (force-stop t))
   (when joint-action-enable
     (prog1 
       (if move-base-trajectory-action
         (and
           (send move-base-action :cancel-all-goals)
           (send move-base-trajectory-action :cancel-all-goals))
         (send move-base-action :cancel-all-goals))
       (if force-stop (send self :go-velocity 0 0 0))))
```
can you give us recommendation? @k-okada